### DISCUSSION TITLE
Enhancement - Rename Rollup app to "Apex Rollup"

### DISCUSSION THREAD

**Discussion by baobao917** [jamessimone/apex-rollup#516] [Ideas]
Here's a small little improvement.  Rename the apex rollup app from "Rollup" to "Apex Rollup". 

Intuitively when searching for the app in app launcher, a user will attempt to search "Apex Rollup"....if they do so, no results will get returned.

**Comment by jamessimone** [comment]
Sure, I can definitely update that

**Reply by baobao917** [reply]
Thanks. I confirmed that v.1.6.0 has this new change - Rename the apex rollup app from "Rollup" to "Apex Rollup".
